language/tools/move-cli/src/main.rs

```diff
@@ -38,9 +41,20 @@ struct Move {
     /// Directory storing Move resources, events, and module bytecodes produced by script execution.
     #[structopt(long, default_value = DEFAULT_STORAGE_DIR, global = true)]
     storage_dir: String,
+    /// Directory storing the depended Move packages.
+    #[structopt(long, default_value = DEFAULT_PACKAGE_DIR, global = true)]
+    package_dir: PathBuf,
```

Comment:
I wouldn't make this an argument right now. No need to be extensible here if we don't need it to be.